core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -160,6 +171,7 @@ abstract class EndToEndAuthorizationTest extends IntegrationTestHarness with Sas
   this.serverConfig.setProperty(KafkaConfig.OffsetsTopicPartitionsProp, "1")
   this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "3")
   this.serverConfig.setProperty(KafkaConfig.MinInSyncReplicasProp, "3")
+  this.serverConfig.setProperty(KafkaConfig.DefaultReplicationFactorProp, "3")
```

Comment:
Why is this needed?

Reply:
because we added tests which use topic auto-create, so they need a default replication factor matching the min in sync